src/Event/EventBundle/Controller/Backend/MediaController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Event\EventBundle\Controller\Backend;
+
+use Symfony\Component\Config\Definition\Exception\Exception;
+use Symfony\Component\HttpFoundation\Request;
+use Event\EventBundle\Controller\Controller;
+use Event\EventBundle\Entity\Media;
+use Event\EventBundle\Form\Type\MediaType;
+
+class MediaController extends Controller
+{
+
+    public function getUploadPath()
+    {
+        return $this->get('kernel')->getRootDir() . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR .
+            'web' . DIRECTORY_SEPARATOR . 'uploads' . DIRECTORY_SEPARATOR . 'media' . DIRECTORY_SEPARATOR;
+    }
+    
+    public function getUploadUrl(){
+        return '/uploads/media/';
+    }
+
+    public function indexAction()
+    {
+        return $this->render('EventEventBundle:Backend/Media:index.html.twig', array(
+            'media' => $this->getRepository('EventEventBundle:Media')->findAll(),
+            'uploadPath' => $this->getUploadPath(),
+            'uploadUrl' => $this->getUploadUrl()
+        ));
+    }
+
+    public function manageAction(Request $request, $id = null)
+    {
+        $oldFileName = '';
+        if ($id === null) {
+            $entity = new Media();
+        } else {
+            $entity = $this->findOr404('EventEventBundle:Media', $id);
+            $oldFileName = $entity->getFilename();
+        }
+
+        $form = $this->createForm(MediaType::class, $entity);
+
+        if ($request->getMethod() === 'POST') {
+            $form->handleRequest($request);
+
+            if ($form->isValid()) {
+                if(is_null($entity->getId())){
```

Comment:
check CS (code style) for all files what you added

`if (` should be with the space, use code sniffer from sensiolab

Reply:
Setup Symfony code sniffer  https://confluence.jetbrains.com/display/PhpStorm/PHP+Code+Sniffer+with+Symfony2+Coding+Standards+Integration+in+PhpStorm+-+Symfony+Development+using+PhpStorm

Reply:
I think this condition in general is excess here